### ISSUE TITLE
[UWP] Action.OpenUrl buttons should appear as links to accessibility

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#3562]
Per a partner team's accessibility review, `Action.OpenUrl` buttons in UWP should be reported as a `hyperlink` type so their behavior lines up with their name.
